README.md

```diff
@@ -1,3 +1,34 @@
 # Sigma v2
 
-Nothing to see here...
+Sigma is a JavaScript library dedicated to graph drawing, mainly developed by [@jacomyal](https://github.com/jacomyal) and [@Yomguithereal](https://github.com/Yomguithereal).
+
+## Overview
+
+[sigmajs.org website](http://sigmajs.org) provides a global overview of the project.
```

Comment:
We should probably state that this link points toward a succinct documentation for v1.

Reply:
done